src/rest_vol.h

```diff
@@ -373,6 +376,17 @@ struct response_buffer {
 };
 extern struct response_buffer response_buffer;
 
+/* Structure that keeps track of semantic version. */
+typedef struct version {
+    size_t major;
+    size_t minor;
+    size_t patch;
+} version;
```

Comment:
I'd probably call this structure something less generic like `server_api_version` so it doesn't get confused among other possible "versions" and this struct should be a part of the `RV_file_t` structure rather than being an extern variable. Otherwise, opening multiple files from different server versions will likely lead to problems.